open-company/open-company-api
# Entry by uuid rather than ?as-of

This PR moves entries from `as-of` to `uuid` identification as comments/reactions (collectively, "interactions") will be referencing the entries and the `as-of` referencing is influenced by older, out of date technical concerns (keeping every entry edit).

https://trello.com/c/uU1gKShp

Turns out to be a no-op against the web UI (great!)

Review the code. NB: Share update requests still spec entries w/ as-of. Share story creation is due to change quite a bit w/ ML so this is left alone for now. All other entry operations are by UUID.

To test, use the Web UI against this branch to:

Start w/ some stuff in your system so you can check the migration.

After the migration, does all your stuff look the same?

Add a new topic
  edit the entry
  add a new entry
  delete the entry
  
Share an update w/ some topics selected

Any other UI regression tests you want to do are good too.
